ext/date/php_date.stub.php

```diff
@@ -377,7 +377,6 @@ public function format(string $format): string {}
 
     /**
      * @tentative-return-type
-     * @alias date_modify
      */
     public function modify(string $modifier): DateTime|false {}
```

Comment:
Hi @derickr, looking at https://3v4l.org/iSPL9,
it doesn't seem possible for 
```
Datetime::modify
DatetimeImmutable::modify
DateInterval::createFromDateString
```
to return false anymore.

Shouldn't the stub be updated then ?

Reply:
@VincentLanglet Good catch, I've just filed https://github.com/php/php-src/pull/14600 fixing these return types.